File: MusicManagerAPI/src/main/java/com/ldz/music/manager/repository/MusicTypeRepository.java
package com.ldz.music.manager.repository;

import com.ldz.music.manager.model.MusicType;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * Created by ldalzotto on 03/05/2017.
 */
public interface MusicTypeRepository extends JpaRepository<MusicType, Long> {
}

File: MusicManagerAPI/src/scala/com/ldz/music/manager/repository/MusicTypeRepository.scala
package com.ldz.music.manager.repository

import com.ldz.music.manager.model.MusicType
import org.springframework.data.jpa.repository.JpaRepository

/**
  * Created by Loic on 20/06/2017.
  */
trait MusicTypeRepository extends JpaRepository[MusicType, java.lang.Long]{

}
